Client/Frontend/Browser/Tabs/State/TabViewState.swift

```diff
@@ -0,0 +1,33 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+struct TabViewState {
+    var isPrivateMode: Bool
+    var tabs: [TabCellState]
+
+    // MARK: Inactive tabs
+    var inactiveTabs: [String]
+    var isInactiveTabsExpanded = true
+
+    var isPrivateTabsEmpty: Bool {
```

Comment:
Should we have unit test for `isPrivateTabsEmpty`? Again here not sure what is mock and what is ready, so asking in case we can add some tests

Reply:
Yes, but there is a separate ticket for private tabs I just move because already existed